CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Note: This release only affects the following runtimes and is not a full system
 ### Fixed
 
 - Kusama People: clear requested judgements that do not have corresponding deposits reserved ([polkadot-fellows/runtimes#339](https://github.com/polkadot-fellows/runtimes/pull/339))
+- Kusama People: adds the calculation of the MetadataHash upon build to be used on CheckMetadata SignedExtension ([polkadot-fellows/runtimes#371](https://github.com/polkadot-fellows/runtimes/pull/371))
```

Comment:
```suggestion
- Kusama People: Build the metadata has at build time, so that `CheckMetadata` can use it at runtime ([polkadot-fellows/runtimes#371](https://github.com/polkadot-fellows/runtimes/pull/371))
```

Reply:
Thanks for the suggestion, I'll change it shortly. Now this will be moved back to 1.2.6 or should it be part of 1.2.7's CHANGELOG?

Reply:
Should be part of the `Unreleased` section only.

Reply:
Yeah added that on my last commit, Joe pointed me on that direction too. Thanks!